tests/EigenTest.cpp

```diff
@@ -0,0 +1,117 @@
+//=============================================================================
+// Copyright (C) 2017-2019 The pmp-library developers
+//
+// This file is part of the Polygon Mesh Processing Library.
+// Distributed under a MIT-style license, see LICENSE.txt for details.
+//
+// SPDX-License-Identifier: MIT-with-employer-disclaimer
+//=============================================================================
+
+#include "gtest/gtest.h"
+
+#include <pmp/SurfaceMesh.h>
+#include <pmp/MatVec.h>
+#include <vector>
+
+using namespace pmp;
+
+class EigenTest: public ::testing::Test
+{
+public:
+
+};
+
+TEST_F(EigenTest, construct_from_eigen)
+{
+    {
```

Comment:
The code blocks in all these tests should be turned into individual test cases. But this can also wait after merge.